movebucket_test.go

```diff
@@ -216,6 +216,129 @@ func TestTx_MoveBucket(t *testing.T) {
 	}
 }
 
+func TestBucket_MoveBucket_DiffDB(t *testing.T) {
```

Comment:
@ahrtr this case must be independent, because i need to create two diff DBs, please correct me if i am wrong